README.md

```diff
@@ -38,6 +37,8 @@ located at:
 https://cc-ama-bot-dev.herokuapp.com/
 
 This is the domain needed to be configured in the Event Subscription of the Slack App Configuration.
+Once Slack App configuration done, go to https://cc-ama-bot-dev.herokuapp.com/login and click Authorize button to
```

Comment:
We should also have a section in the readme that says we're using botkit (and maybe a link to the docs)

Reply:
> We should also have a section in the readme that says we're using botkit (and maybe a link to the docs)

will do that in this branch.